Misc/NEWS.d/next/Library/2018-09-01-20-43-10.bpo-34563.7NQK7B.rst

```diff
@@ -0,0 +1 @@
+Fix for invalid assert on big output of multiprocessing.Process
```

Comment:
You should mention _winapi.PeekNamedPipe() and _winapi.ReadFile(). Maybe:

> Fix _winapi.PeekNamedPipe() and _winapi.ReadFile() for read larger than INT_MAX (usually 2^31-1).

I'm not sure that it's worth it to mention multiprocessing in the NEWS entry.

Reply:
`_winapi` is a private module. `_winapi.PeekNamedPipe()` and `_winapi.ReadFile()` are used only in `multiprocessing.connection`. It is an implementation detail. The end user uses `multiprocessing`.